docs/interactions/context_menus.rst

```diff
@@ -0,0 +1,50 @@
+.. currentmodule:: discord
+
+.. _guide_interactions_context-menus:
+
+Context Menus
+==============
+
+Context menus allow for commands to be triggered through a ``context menu`` upon right clicking a related object, then selecting the name of the command to run from the ``Apps`` menu.
+
+- Context menus can be categorized into two separate types, one for :class:`.User` and :class:`.Member` objects, and one for :class:`.Message` objects.
+- The command does not have any arguments, and will simply return the target object.
```

Comment:
This seems out of place for a prose. It might make more sense to restructure this to refer to the discord UI for the prose, before diving into the code details later on.